src/main/java/com/sprints/validations/SprintsValidations.java

```diff
@@ -46,4 +46,11 @@ public void sprintsActiveValidation(Optional<Sprint> sprintOptional){
 			throw new EntityConflictException("You cannot delete an active sprint", "/");
 		}
 	}
+	
+	public void sprintValidateStartDate(SprintDomain sprintFinal){
+		
+		if((sprintFinal.getStart_date().isAfter(sprintFinal.getEnd_date())) || (sprintFinal.getStart_date().isEqual(sprintFinal.getEnd_date()))) {
```

Comment:
what happen if start date or end date is null, or empty ?

Reply:
Ok, there are a couple of things that you need change, because with this validation you are doing mandatory 2 fields that should be empty